paper/paper.md

```diff
@@ -0,0 +1,174 @@
+    index: 2
+  - name: Department Heterogeneous Reactions, Max Planck Institute for Chemical Energy Conversion, Mülheim an der Ruhr, Germany
+    index: 3
+  - name: Department of Physical Chemistry, Fritz Haber Institute of the Max Planck Society, Berlin, DE
+    index: 4
+  - name: Felix-Bloch-Institut für Festkörperphysik, Universität Leipzig, Leipzig, Germany
+    index: 5
+date: 06 June 2025
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Scientific data across experimental physics and materials science often lacks adherence to FAIR principles [@Wilkinson:2016; @Jacobsen:2020; @Barker:2022] due to incompatible instrument-specific formats and diverse standardization practices. pynxtools is a Python software development framework with a CLI interface that standardizes data conversion for scientific experiments in materials characterization to the NeXus format [@Koennecke:2015] across diverse scientific domains. NeXus uses NeXus application definitions as their data storage specifications. pynxtools provides a fixed, versioned set of NeXus application definitions that ensures convergence and alignment in data specifications across photoemission spectroscopy, electron microscopy [@Prestat:2025], atom probe tomography, optical spectroscopy, scanning probe microscopy, and X-ray diffraction. Through its modular plugin architecture, pynxtools provides maps for instrument-specific raw data, and electronic lab notebook metadata, to these unified definitions, while performing validation to ensure data correctness and NeXus compliance. By simplifying the adoption of standardized application definitions, the framework enables true data interoperability and FAIR data management across multiple experimental techniques.
+
+# Statement of need
+
+Achieving FAIR (Findable, Accessible, Interoperable, and Reproducible) data principles in experimental physics and materials science requires consistent implementation of standardized data formats. While NeXus provides comprehensive data specifications for structured storage of scientific data, pynxtools simplifies the implementation process for developers and researchers by providing guided workflows and automated validation to ensure complete compliance. Existing tools [@Koennecke:2024; @Jemian:2025] provide solutions with individual capabilities, but none offers a comprehensive end-to-end workflow for proper NeXus adoption. pynxtools addresses this critical gap by providing an accessible framework that enforces complete NeXus application definition compliance through automated validation, detailed error reporting for missing required data points, and clear implementation pathways via configuration files and extensible plugins. This approach transforms NeXus from a complex specification into a practical solution, enabling researchers to achieve true data interoperability without deep technical expertise in the underlying standards.
```

Comment:
The jump from "standardized data formats" to "While NeXus" is too large. If there are several cross-community standards we should mention these and then state like "one is NeXus". Then we can continue with the next sentence.

The sentence "While ..., pynxtools" to me is a contrast where "pynxtools simplifies" has been injected right in the middle but where I would rather expect first an acknowledging statement like 
"NeXus provides comprehensive data specifications for structured storage of scientific data" but followed by a statement what the challenges are namely that "Existing tools provide .... but for proper NeXus adoption". 

Then one can start with "pynxtools addresses and simplifies this ... , and clear implementation pathways ... and extensible plugins."